### PR TITLE
add: create target_performance data

### PR DESCRIPTION
**Pull request by Spherrrical**
# Title

## Discord username

Spherical#0001
## Changes you have made

Added a performance cfg file with stall data.
## References used

https://contentzone.eurocontrol.int/aircraftperformance/details.aspx?ICAO=B77W
## Additional comments

**QA testing coming soon**

### CODE REVIEW

File: PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+empty_weight_pitch_MOI = 125000000 ; Empty pitch moment of inertia, Jxx (SLUG SQ FEET)
+empty_weight_roll_MOI = 75000000 ; Empty roll moment of inertia, Jzz (SLUG SQ FEET)
+empty_weight_roll_MOI-pctJxx = 0.6; 
+empty_weight_yaw_MOI = 125000000 ; Empty yaw moment of inertia, Jyy (SLUG SQ FEET)
+empty_weight_yaw_MOI-pctJxx = 1.00; 
+empty_weight_coupled_MOI = 0.0 ; Empty transverse moment of inertia, Jyz (SLUG SQ FEET)
+mpty_weight_coupled_MOI-pctJxx = 0.;
+
+[DOUBLE_CHECKS]
+wing_area = 4998,0.1;
```
Comment (scoopscode) [inline PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:59]: What units is this in?

Comment (Spherrrical) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:59]: Wait...I need to fix this.

Comment (Spherrrical) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg]: This should be in feet.

File: PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+[DOUBLE_CHECKS]
+wing_area = 199,0.1;
+wing_span = 212.43,0.1;
+htail_area = 800,0.1;
+htail_span = 70,0.1;
+vtail_area = 800,0.1;
+vtail_span = 33,0.1;
+max_gross_weight = 698645,1.0;
+empty_weight = 220000,1.0;
+max_hp = 37828,0.1;
```
Comment (scoopscode) [inline PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:67]: I believe this is 55,000-60,000 hp.

**Review by scoopscode — CHANGES_REQUESTED** [review]

File: PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg (hunk 2 above)
Comment (Spherrrical) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:67]: Resolved.

Comment (scoopscode) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:67]: Is this indicating 37,828 hp per engine or for the entire aircraft?

Comment (Spherrrical) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:67]: Entire aircraft.

Comment (scoopscode) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:67]: Pretty sure one engine does around 40,000 hp. Might wanna check your stats.

Comment (scoopscode) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg:67]: @Spherrrical ...

**Comment by scoopscode** [comment]
Since @Spherrrical seems to not really care about this PR, I will be closing it and wait for him to resolve/fix requested changes

**Comment by scoopscode** [comment]
@Spherrrical check #8

File: PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg (hunk 2 above)
Comment (Spherrrical) [reply PinpointB777/PackageSources/SimObjects/Airplanes/Pinpoint_B777/target_performance.cfg]: Added it @scoopscode

**Comment by scoopscode** [comment]
Ok strange I thought I opened one today for this but let me review

**Comment by scoopscode** [comment]
LGTM!

**Pull request merged**